plugin/audit_log/tests/mtr/audit_log_default_db.test

```diff
@@ -47,7 +49,7 @@ connection test;
 SELECT * FROM t;
 use 	`db1`;
 --error ER_DBACCESS_DENIED_ERROR
-change_user user2,111,db1;
+change_user user2,111,db1,false;
```

Comment:
Maybe "reconnect_on_fail, do_not_reconnect_on_fail" or similar so that it's obvious what does "false" mean here?